generators/client/templates/react/src/main/webapp/app/config/icon-loader.ts.ejs

```diff
@@ -76,6 +76,8 @@ export const loadIcons = () => {
     faFlag,
     faBell,
     faHome,
+    faRoad,
```

Comment:
isn't this duplicate of last entry?

Reply:
It is, I meant to add `faSearch`. :sweat_smile: